.github/workflows/create-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Create deployment
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+    types:
+      - labeled
+env:
+  GITOPS_ENVIRONMENT: stage
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy_nodes')
+
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+
+      - name: Install Jinja2 cli
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
This just takes extra time, but has no value, right?

Reply:
I think the same, just took it from an example claiming that was suggested as per security reasons, but I suppose that Github will take care of that and we don't have to do it again.
I've removed as well.